test/Test/Pos/UtilSpec.hs

```diff
@@ -41,6 +66,22 @@ spec = describe "Util" $ do
         \ corresponding to these keys have a non-empty intersection, the difference\
         \ map's inner maps corresponding to those keys will be smaller in size than the\
         \ inner maps in the minuend hashmap"
+    description_One =
+        "Turning a single element into a chronological is the same as turning into the\
+        \ structure within and applying the outer data constructor."
+    description_toFromListNew functor =
+        "Converting 'NewestFirst " ++ functor ++ " a' to '[Item a]' and back changes\
```

Comment:
Perhaps using `formatToString` by analogy to `sformat` will be more convinient here